src/backend/src/services/change-requests.services.ts

```diff
@@ -445,21 +450,25 @@ export default class ChangeRequestsService {
           include: {
             workPackage: {
               include: {
-                project: { include: { team: true, wbsElement: true } }
+                project: { include: { teams: true, wbsElement: true } }
               }
             }
           }
         }
       }
     });
 
-    const team = createdChangeRequest.wbsElement.workPackage?.project.team;
-    if (team) {
+    const teams = createdChangeRequest.wbsElement.workPackage?.project.teams;
+    if (!teams) {
+      throw new HttpException(400, 'This project needs to be assigned to a team to create a task!');
```

Comment:
same as comment above, this is for creating a stage gate CR, and we should do it how it was done before